### PR TITLE
impr(quotes): add English quotes (TheExploration)

### PR DESCRIPTION
**Pull request by TheExploration**
### Description

Add 2 English quotes

### Checks

- [x] Adding quotes?
  - [x] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [x] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38682,6 +38682,18 @@
       "source": "Henry David Thoreau",
       "length": 78,
       "id": 7036
+    },
+    {
+      "text": "The very diction. An ever growing undertaking of dreams. Stare out over the expanse ++ WONDER",
+      "source": "Antrum",
+      "length": 93,
+      "id": 7037
+    },
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38691]: Please remove, seems random.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/english.json (hunk 1 above)
Comment (TheExploration) [reply frontend/static/quotes/english.json]: I like this one but yeah, its not a traditional quote

**Review by TheExploration — COMMENTED** [review]
Removed quote

**Comment by Miodec** [comment]
Thanks

**Pull request merged**